docs/getting-started/index.asciidoc

```diff
@@ -7,31 +7,38 @@ Looking to get started with {elastic-sec}? This section describes the {elastic-s
 TIP: View the https://www.elastic.co/training/elastic-security-quick-start[{elastic-sec} Quick Start video] to learn how to configure your endpoints with {elastic-sec} so you can stream, detect, and visualize threats in real time on {ecloud}.
 
 include::sec-app-requirements.asciidoc[leveloffset=+1]
-include::detections-req.asciidoc[leveloffset=+2]
-include::cases-req.asciidoc[leveloffset=+2]
-include::ers-req.asciidoc[leveloffset=+2]
-include::ml-req.asciidoc[leveloffset=+2]
 include::defend-feature-privs.asciidoc[leveloffset=+2]
-include::net-map-req.asciidoc[leveloffset=+2]
+
+include::security-ui.asciidoc[leveloffset=+1]
+
 include::endgame-sensor-reqs.asciidoc[leveloffset=+1]
 include::endgame-sensor-FDA.asciidoc[leveloffset=+2]
 include::endgame-sensor-FDA-ven.asciidoc[leveloffset=+2]
+
 include::security-spaces.asciidoc[leveloffset=+1]
+
 include::data-views-in-sec.asciidoc[leveloffset=+1]
+
 include::ingest-data.asciidoc[leveloffset=+1]
```

Comment:
I would put ingest after `Elastic Endgame reqs`, closer to the top. Here's the proposed order according to our [doc](https://docs.google.com/document/d/1YlhTZRWjx__Y2-5R-ndiq9mgc_ZejGD3RcHbXzrfYoQ/edit?usp=sharing) but I'm open to suggestions.  

![2024-07-25_14-49-41](https://github.com/user-attachments/assets/091c7f96-0083-4e97-9e99-e9d51e038d83)

Reply:
"does this PR move the feature-specific reqs to their respective sections?" - Yes, the only exception being the Elastic Defend stuff since we haven't made a section for it yet.